kDrive/UI/View/Files/FloatingPanel/FloatingPanelQuickActionCollectionViewCell.swift

```diff
@@ -114,6 +115,7 @@ class FloatingPanelQuickActionCollectionViewCell: UICollectionViewCell {
 
     #if !ISEXTENSION
     func configureDownload(with file: File, action: FloatingPanelAction, progress: CGFloat?) {
+        @LazyInjectService var downloadQueue: DownloadQueueable
```

Comment:
Since we are in a function, we do not need the `lazy` aspect of the DI here.
```suggestion
        @InjectService var downloadQueue: DownloadQueueable
```